pkg/tnf/interactive/spawner.go

```diff
@@ -272,6 +281,23 @@ func NewGoExpectSpawner() *GoExpectSpawner {
 	return &GoExpectSpawner{}
 }
 
+// LogStdErr logs stderr output to logger (warning)
+func LogStdErr(cmdLine string, stderrpipe io.Reader) {
+	buf := bufio.NewReader(stderrpipe)
+
+	for {
+		line, _, err := buf.ReadLine()
+		log.Warn("STDERR for " + cmdLine + " : " + string(line))
+
+		if err != nil {
+			// Some Error has happened, goroutine about to exit
+			log.Warn(err)
+			return
+		}
+		time.Sleep(100 * time.Millisecond) //nolint:gomnd //100 ms variable used only here
```

Comment:
Is this sleep really necessary? The buf.ReadLine() seems to be a blocking call.

Reply:
fixed